README.md

```diff
@@ -1,240 +1,268 @@
 
+1. Copy `env_blank.json`.
+1. Rename the file `env.json` (for use in local development) or `[some name].json` (for use in OpenShift).
+1. Replace the empty strings and `0`'s with the desired settings and values.
 
-### Docker
+The meanings of the keys and their expected values are described in the table below.
 
-#### Running kartograafr in  Docker locally
+**Key** | **Description**
+----- | -----
+`Logging_Level` | The minimum level for log messages that will appear in output. `INFO` or `DEBUG` is recommended for most use cases; see [Python's logging module](https://docs.python.org/3/library/logging.html).
+`Logging_Directory` | The path where log files will be written (see **OpenShift** under Installation & Development for a description of an OpenShift-related restriction on this value).
+`SMTP_Server` | The name of the server emails should be sent from, if the `--email` flag is in use.
+`Email_Sender_Address` | The name and email address to use in the FROM header of emails sent. This should take the form of "\\"Severus Snape\\" <halfbloodprince@umich.edu>" (make sure to escape double quotes).
+`Canvas_Base_URL` | The URL of the Canvas instance you want to pull Canvas data from; for production at UM, this is `umich.instructure.com`.
+`Canvas_API_Token` | The API token to use when making requests for data related to courses, assignments, and users.
+`Canvas_Config_Course_ID` | The ID number of the configuration course, (see the **Canvas** Configuration section above).
+`ArcGIS_Org_Name` | The name of the ArcGIS organization in use.
+`ArcGIS_Username` | The name of an arcGIS user with permission for creating and modifying user groups.
+`ArcGIS_Passowrd` | The name of the password for the username provided above.
 
-The *runDocker.sh* script will configure and build a local Docker
-container for kartograafr and will setup a Docker environment that
-allows running that container on OSX. This script  is not needed when running
-on OpenShift since that provides a rich environment of services.
-A Dockerfile is provided for local development. It assumes that the
-code has been checked out already.  
+----------------
 
-To run on a local Docker environment, use
+## <a name='installationAndDevelopment'></a>Installation & Development
+
+The sections below provide instructions for installing and running the application in various environments. Before
+attempting to install or deploy, ensure the steps in the **For Administrators & Developers** section in 
+Configuration have been completed. Depending on the environment you plan to run the application in, you may also need 
+to install some or all of the following:
+   * Python
+   * Git
+   * [Docker Desktop](https://www.docker.com/products/docker-desktop)
+   * [OpenShift CLI](https://docs.openshift.com/enterprise/3.1/cli_reference/get_started_cli.html)
+
+### Local Development
+
+To set up the application for use locally or in preparation for development work, do the following:
```

Comment:
I would like to suggest a reconstruction of the following sections into:

* Sending emails settings
* Deployments 
    * Local Development
        * with Docker (preferred)
        * with virtual environment
    * OpenShift Deployment

"Send Email" should be in another separate section from the deployment choices, since it is common task, independent of deployment environments.

Reply:
Thanks for the idea. In my latest commits, I've tried to mostly go down this route. I opted to put the Sending Email section after, since I feel the other explanations need to come first. It's also a little awkward, since we don't have a great way to use Docker locally with the mock email server; I just documented this in issue #47. Let me know what you think!